### PR TITLE
[:fix] add mobile option, fix useOnClickOutside hook, hover disabled on touchdevices 

### PR DESCRIPTION
**Pull request by svenvoskamp**
add mobile option, fix useOnClickOutside hook, hover disabled on touchdevices 

# Description

* No extra tap required to open push overview. 
* Refactored `useOnClickOutside`. 
* Some hover states disabled on mobile

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Fixes/Resolves (Optional)

Resolves https://github.com/WalletConnect/web3inbox/issues/154

# Examples/Screenshots (Optional)

Please test on a mobile device: 

https://web3inbox-dev-hidden-git-fix-push-tap-walletconnect1.vercel.app/

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package-lock.json
Comment (devceline) [inline package-lock.json:1]: We don't need a package-lock, we use `yarn`

File: yarn.lock
Comment (devceline) [inline yarn.lock:1]: Why delete `yarn.lock`?

**Review by devceline — CHANGES_REQUESTED** [review]
The actual changes look good, but please stick to `yarn`

**Comment by svenvoskamp** [comment]
@devceline 

Can you please review again? 

* I recovered the `yarn.lock` file
* Adjusted the clear/unread animation on mobile 
* Small UX improvements desktop/mobile notifications settings toggle

File: src/utils/hooks.ts [hunk 1]
```diff
@@ -38,12 +38,18 @@ export const useOnClickOutside = (
...
+    }
 
     return () => {
-      document.removeEventListener('mousedown', listener)
-      document.removeEventListener('touchstart', listener)
+      if (isMobile()) {
+        document.removeEventListener('touchstart', listener)
+      } else {
+        document.removeEventListener('mousedown', listener)
+      }
```
Comment (devceline) [inline src/utils/hooks.ts:52]: Could be simplified into

````ts
const event = isMobile()? `touchstart` : `mousedown`

document.addEventListener(event, listener)

return () => document.removeEventListener(event, listener)
```

**Review by devceline — APPROVED** [review]
LGTM, one minor suggestion

**Pull request merged**